feature/experimental/interface/my_station_mac/ate_tests/my_station_mac_test/my_station_mac_test.go

```diff
@@ -0,0 +1,335 @@
+	clientA.AddIPv4(t, ateDstNetCIDR, nhgIndex, *deviations.DefaultNetworkInstance, "", fluent.InstalledInRIB)
+
+	t.Logf("Verify through AFT Telemetry that %s is active", ateDstNetCIDR)
+	ipv4Path := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Afts().Ipv4Entry(ateDstNetCIDR)
+	if got, ok := gnmi.Watch(t, dut, ipv4Path.Prefix().State(), time.Minute, func(val *ygnmi.Value[string]) bool {
+		prefix, present := val.Val()
+		return present && prefix == ateDstNetCIDR
+	}).Await(t); !ok {
+		t.Errorf("ipv4-entry/state/prefix got %v, want %s", got, ateDstNetCIDR)
+	}
+}
+
+// testTraffic generates and verifies traffic flow with destination MAC as my station MAC
+func testTraffic(
+	t *testing.T,
+	ate *ondatra.ATEDevice,
+	top *ondatra.ATETopology,
+	headers ...ondatra.Header,
+) {
+	i1 := top.Interfaces()[ateSrc.Name]
+	i2 := top.Interfaces()[ateDst.Name]
+
+	// Egress tracking inspects packets from DUT and key the flow
+	// counters by custom bit offset and width.  Width is limited to
+	// 15-bits.
+	//
+	// Ethernet header:
+	//   - Destination MAC (6 octets)
+	//   - Source MAC (6 octets)
+	//   - Optional 802.1q VLAN tag (4 octets)
+	//   - Frame size (2 octets)
+	//
+	// myStationMAC is "02:12:34:56:78:1a";
+	// last 15-bits of myStationMAC is 0x781a = 30746
+
+	stationMacFilterVal := "30746"
+
+	flow := ate.Traffic().NewFlow("Flow").
+		WithSrcEndpoints(i1).
+		WithDstEndpoints(i2).
+		WithHeaders(headers...)
+	flow.EgressTracking().WithOffset(33).WithWidth(15)
+
+	ate.Traffic().Start(t, flow)
+	time.Sleep(10 * time.Second)
+	ate.Traffic().Stop(t)
+
+	flowPath := gnmi.OC().Flow(flow.Name())
+	inPkts := gnmi.Get(t, ate, flowPath.Counters().InPkts().State())
+
+	if got := gnmi.Get(t, ate, flowPath.LossPct().State()); got > 0 {
+		t.Errorf("LossPct for flow %s got %g, want 0", flow.Name(), got)
+	}
+
+	etPath := flowPath.EgressTrackingAny()
+	ets := gnmi.GetAll(t, ate, etPath.State())
+	for i, et := range ets {
+		fptest.LogQuery(t, fmt.Sprintf("ATE flow EgressTracking[%d]", i), etPath.State(), et)
+	}
+
+	if got := len(ets); got != 1 {
+		t.Errorf("EgressTracking got %d items, want 1", got)
+		return
+	}
+
+	if got := ets[0].GetFilter(); got != stationMacFilterVal {
+		t.Errorf("EgressTracking filter got %q, want %q", got, stationMacFilterVal)
+	}
+
+	if got := ets[0].GetCounters().GetInPkts(); got != inPkts {
+		t.Errorf("EgressTracking counter in-pkts got %d, want %d", got, inPkts)
+	}
+}
+
+func TestMyStationMAC(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	t.Logf("Configure DUT")
+	configureDUT(t)
+
+	t.Logf("Configure ATE")
+	ate := ondatra.ATE(t, "ate")
+	top := configureATE(t, ate)
+	top.Push(t).StartProtocols(t)
+
+	t.Logf("Configure My Station MAC")
+	gnmi.Replace(t, dut, gnmi.OC().System().MacAddress().RoutingMac().Config(), myStationMAC)
+
+	t.Logf("Configure static ARP on DUT for ate:port2 so the destination MAC to ate:port2 is also the My Station MAC")
+	configStaticArp(t)
```

Comment:
can we delete the configured static arp/ipv6 nd entries while existing the test. This would allow rerun of the test without manual intervention. Else protocols will not comeup on ate and test would fail on a rerun because of stale configs.

Reply:
@singh-prem : Thanks for reviewing the script.

Deletion of previously configured static arp/v6 ND entries is taken care by 'Replace' action at https://github.com/openconfig/featureprofiles/blob/39171fb5871430956f20824cd4da2a77a238d88e/feature/experimental/interface/my_station_mac/ate_tests/my_station_mac_test/my_station_mac_test.go#L138. 
as OC path for static arp/ v6 ND is:
/interfaces/interface/subinterfaces/subinterface/ipv4/neighbors/neighbor/config/link-layer-address
/interfaces/interface/subinterfaces/subinterface/ipv6/neighbors/neighbor/config/link-layer-address

Please let me know if I'm missing any specific case where entries wont be deleted?

Reply:
No static arp config needed according to latest testplan (v10). So removed this code. Thanks.